azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+name: tl-shared-service
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+    clean: all
+
+variables:
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+pool:
```

Comment:
Change this to use one of our agent pools?

Reply:
No these don't have access to CIP